backend/exchanges/pocket.go

```diff
@@ -0,0 +1,34 @@
+package exchanges
+
+import (
+	"fmt"
+
+	"github.com/digitalbitbox/bitbox-wallet-app/backend/accounts"
+	"github.com/digitalbitbox/bitbox-wallet-app/backend/coins/coin"
+)
+
+const (
+	// pocketAPITestURL is the url of the pocket widget in test environment.
+	//pocketAPITestURL = "http://widget.staging.pocketbitcoin.com/widget_mjxWDmSUkMvdQdXDCeHrjC"
+
+	// pocketAPILiveURL is the url of the pocket widget in production environment.
+	pocketAPILiveURL = "http://widget.pocketbitcoin.com/widget_vqx25E6kzvGBYGjN2QoXVH"
+)
+
+// BuyPocket verifies if the passed account is enabled for Pocket and returns the
+// url needed to incorporate the widget in the frontend.
+func BuyPocket(acct accounts.Interface) (string, error) {
+	if acct.Coin().Code() != coin.CodeBTC {
+		return "", fmt.Errorf("unsupported cryptocurrency code %q", acct.Coin().Code())
+	}
+
```

Comment:
Use `IsPocketSupported()`?

is this check even needed? in the frontend this call doesn't happen unless you selected pocket (where the check already happened).

Reply:
Right. Slightly modified the function to add TBTC support, as you proposed here https://github.com/digitalbitbox/bitbox-wallet-app/pull/1846#discussion_r1023795009